Dockerfile

```
@@ -1,6 +1,16 @@
 FROM node:10.15.3-alpine@sha256:313c5c88acc0ec12a9abca9a83719065dfd54e94ffc56464b7ce24998dd2838d
+
+LABEL repository="https://github.com/toolmantim/release-drafter"
+LABEL homepage="https://github.com/toolmantim/release-drafter"
+LABEL maintainer="Tim Lucas"
+
+LABEL com.github.actions.name="Release Drafter"
+LABEL com.github.actions.description="Drafts your next release notes as pull requests are merged into master."
+LABEL com.github.actions.icon="edit-2"
+LABEL com.github.actions.color="orange"
+
 WORKDIR /src
 ADD     yarn.lock package.json /src/
 RUN     yarn
 ADD     . /src
-CMD     ["npm", "start"]
+CMD     ["npm", "receive"]
```

I tried using the action in one of my repos, I got an error about `npm receive`. I think it should be:

```suggestion
CMD     ["npm", "run", "receive"]
```

---

Good catch! I made that change, let me know if it works.

---

`npm run receive` seems to work now :)
I'm now getting the following error:

```
docker run image=release-drafter:add-github-action-support entrypoint=[] cmd=[]
npm ERR! path /github/workspace/package.json
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open '/github/workspace/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /github/home/.npm/_logs/2019-04-26T13_12_48_443Z-debug.log
```

It's possible to debug locally using [act](https://github.com/nektos/act).

---

Ah, looks like we can’t use `WORKDIR`, which finally explains to me why all the GH actions focus on entrypoints.

I hadn’t quite gotten to the point of testing this on another repo with act! You’re one step ahead. Act is awesome tho. I used it for dev’ing https://github.com/buildkite/trigger-pipeline-action

---

If you drop the workdir and use ./ In the add you should be fine

---

You guys know much more about Docker than me probably, but maybe we can take this as an example: https://github.com/JasonEtco/todo/blob/dockerfile/Dockerfile? :)

---

All fixed!

---

Nice! 😄

---

Strangely, the `branches` key in the `release-drafter` config doesn't seem to be working as expected. But it should give you a chance to test it @TimonVS!

---

Seems to work for me now! I didn't do very extensive testing though, but we can start using it on this repo before we publish it to the marketplace to see if everything works nicely? I will also start using the Action on the repos at work starting on Monday.